include/openrave/environment.h

```diff
@@ -23,23 +23,16 @@
 #define  OPENRAVE_ENVIRONMENTBASE_H
 
 #include <openrave/config.h>
+#include <mutex>
 
 namespace OpenRAVE {
 
 #if OPENRAVE_ENVIRONMENT_RECURSIVE_LOCK
-#if OPENRAVE_STD_SCOPED_LOCK
-#include <mutex>
 using EnvironmentMutex = ::std::recursive_mutex;
-using EnvironmentLock  = ::std::scoped_lock<std::recursive_mutex>;
+using EnvironmentLock  = ::std::unique_lock<std::recursive_mutex>;
```

Comment:
Is there any perf difference between scoped_lock and unique_lock? Or is it just decorative for compilation of c++17?